resources/companies/wt.csv

```diff
@@ -0,0 +1,94 @@
+"Kod kraju","Koordynaty","Adres","Nazwa firmy","Miejscowość","Kierunek","Tagi","Strona internetowa","Email","Telefon","Płatne?"
+"pl","51.20872689999999,16.1637251","Najświętszej Marii Panny 5F/5, 59-220 Legnica, Poland","Blumilk sp. z o.o.","Legnica","Informatyka","Programowanie, Web, PHP, Laravel, Vue","https://www.blumilk.pl/","hello@blumilk.pl",,"Nie"
+"pl","50.9456692,14.9081164","Młodych Energetyków 12, 59-916 Bogatynia, Poland","PGE Elektrownia Turów Spółka Akcyjna","Bogatynia","Informatyka","IT","https://elturow.pgegiek.pl/"," elektrowniaturow.giekelt@gkpge.pl","75 773 49 00","Nie"
+"pl","50.92941889999999,14.9247144","Górników Turowa 1, 59-916 Bogatynia, Poland","BOT Kopalnia Węgla Brunatnego Turów SA","Bogatynia","Informatyka","IT","https://kwbturow.pgegiek.pl/","kopalnia.giekkwbt@gkpge.pl","75 77 35 300","Nie"
```

Comment:
`BOT` holding doesn't exist since 2007
https://pl.wikipedia.org/wiki/BOT_(holding)
```suggestion
"pl","50.92941889999999,14.9247144","Górników Turowa 1, 59-916 Bogatynia, Poland","PGE Kopalnia Węgla Brunatnego Turów SA","Bogatynia","Informatyka","IT","https://kwbturow.pgegiek.pl/","kopalnia.giekkwbt@gkpge.pl","75 77 35 300","Nie"
```